src/main/scala/temple/generate/server/go/common/GoCommonGenerator.scala

```diff
@@ -42,4 +42,43 @@ object GoCommonGenerator {
         ),
       ),
     )
+
+  /** Generate declaration and assignment statement */
+  private[go] def genDeclareAndAssign(expr: String, identifiers: String*): String =
+    mkCode.list(identifiers) + " := " + expr // mkCode doesn't put a space before the colon
```

Comment:
Could do `mkCode(mkCode.list(identifiers), " :=", expr)`, but you’re right, mkCode doesn’t add a space because it’s rare for colons to be preceded by a space. Not sure how to make it nicer though

Reply:
I had that, but figured then the outer mkCode wasn't really doing it's job

Reply:
Don't think it's particularly necessary to fix